example_scenes/manim_jupyter_example.ipynb

```diff
@@ -0,0 +1,63 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from manim import *"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%manim Example1 -s -p -ql -v WARNING --disable_caching\n",
```

Comment:
We should make this PR compliant with the imminent changes from #1013, the scene name has to be the last argument that is passed.

```suggestion
    "%%manim -s -p -ql -v WARNING Example1\n",
```